packages/venia-concept/src/RootComponents/Category/category.js

```diff
@@ -54,16 +58,18 @@ class Category extends Component {
             totalPages: prevPageTotal
         };
 
+        const queryVariables = {
+            id: Number(id),
+            onServer: false,
+            pageSize: Number(pageSize),
+            currentPage: Number(currentPage),
+            idString: String(id)
+        };
+
+        const isFilterModalOpen = drawer === 'filter';
```

Comment:
This is what I mean by not having to thread the prop down. Infact in the future if some other form of page besides a `category` page wants to apply some filters it will just need to use the `toggleDrawer('filter')` action and not need to pass this prop down.

Reply:
Fixed in latest commit